gitlab.go

```diff
@@ -60,6 +60,7 @@ const (
 	basicAuth authType = iota
 	oAuthToken
 	privateToken
+	jobToken
```

Comment:
Please place this in the correct alphabetical order to keep things consistent.

Reply:
I put it in this order in order to not introduce potential backwards incompatibility. Although, it's an unexported field, I thought, it could be dangerous to put it before. But as you think it's fine, I will change it.